src/api/api.cc

```diff
@@ -11071,7 +11071,8 @@ static std::set<std::string>* gRecordReplayKnownFeatures = new std::set<std::str
 // Ideally, this should always be a short list.
 // NOTE: These should generally be "double-negative" flags which we need to convert to positive in the near future.
 static const char* gExperimentalFlags[] = {
-  "disable-collect-events"
+  // NOTE: Events are currently in the pipeline, but causing too many crashes. Re-enable once ready.
+  // "disable-collect-events"
```

Comment:
Wait, why isn't commenting out "disable-collect-events" the same as enabling them?  Isn't this enabling event collection?

Reply:
All flags are enabled by default. Adding them to this set disables them. Double negative ftw!